android-patches/patches/Build/ReactAndroid/packages.config

```diff
@@ -1,5 +1,8 @@
---- /dev/null	2022-01-12 17:14:59.000000000 -0800
```

Comment:
As discussed, we should incorporate this new diff command to the tool so that we can use the tool to generate and apply the diff/new file patches..

Reply:
Created AB#5847138 to track this work